src/sd/sd_server.cpp

```
@@ -228,6 +233,7 @@ class SdServerImpl : public transport::ITransportListener {
         multicast_option->set_port(someip_htons(config_.multicast_port));
         response_message.add_option(std::move(multicast_option));
 
+        // NOLINTNEXTLINE(cppcoreguidelines-pro-type-static-cast-downcast)
```

Cppcheck (2.20.0)

[style] The function 'set_reboot' is never used.

(unusedFunction)